src/app.d.ts

```diff
@@ -2,7 +2,7 @@
 
 declare namespace App {
   interface Locals {
-    permissibleQueries: Record<string, true> | null;
+    permissibleQueries: import('./types/permissions').PermissibleQueriesMap | null;
```

Comment:
Know this is following the conventions set below but curious if you know why these imports are declared like this and not at the top?

Reply:
In a `.d.ts` file this is the only way to get it correctly resolving afaik.